packages/expo-modules-autolinking/src/ExpoModuleConfig.ts

```diff
@@ -17,52 +22,67 @@ export class ExpoModuleConfig {
    * Whether the module supports given platform.
    */
   supportsPlatform(platform: SupportedPlatform): boolean {
-    return this.rawConfig.platforms?.includes(platform) ?? false;
+    const supportedPlatforms = this.rawConfig.platforms ?? [];
+
+    if (platform === 'apple') {
+      // Apple platform is supported when any of iOS, macOS and tvOS is supported.
+      return supportedPlatforms.some((supportedPlatform) => {
+        return ['apple', 'ios', 'macos', 'tvos'].includes(supportedPlatform);
+      });
+    }
```

Comment:
opened the diff looking for this specifically :+1: good to ensure that there is compat with existing configs